sdk/src/types/block/slot/commitment.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crypto::hashes::{blake2b::Blake2b256, Digest};
+use packable::{packer::SlicePacker, Packable, PackableExt};
+
+use crate::types::block::slot::{RootsId, SlotCommitmentId, SlotIndex};
+
+/// Contains a summary of a slot.
+/// It is linked to the commitment of the previous slot, which forms a commitment chain.
+#[derive(Clone, Debug, Eq, PartialEq, Hash, derive_more::From, Packable)]
```

Comment:
can this be `Copy` even?

Reply:
Sure, but should it? It's not super small

Reply:
changed my mind